### ISSUE TITLE
[SplitButton][Accessibility] SplitButton dropdown button is not accessible

### ISSUE THREAD

**Issue by dtopalov** [telerik/kendo-angular#2139]
The WAVE accessibility assessment tool reports an issue related to the SplitButton's dropdown button being empty:

![image](https://user-images.githubusercontent.com/7533599/54759210-47ca7780-4bf6-11e9-910b-c477c4d849ba.png)

**Errors**
Empty button
**What It Means**
A button is empty or has no value text.
**Why It Matters**
When navigating to a button, descriptive text must be presented to screen reader users to indicate the function of the button.
**How to Fix It**
Place text content within the <button> element or give the <input> element a value attribute.
The Algorithm... in English
A <button> element is present that contains no text content (or alternative text), or an <input type="submit">, <input type="button">, or <input type="reset"> has an empty or missing value attribute.
**Standards and Guidelines**
[1.1.1 Non-text Content (Level A)](https://webaim.org/standards/wcag/checklist#sc1.1.1)
[2.4.4 Link Purpose (In Context) (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.4)

**Comment by PetarMetodiev** [comment]
According to the [docs](https://www.telerik.com/kendo-angular-ui-develop/components/buttons/splitbutton/keyboard-navigation/) the SplitButton is opened using `Alt` + `DownArrow`. When using `Tab` for navigation, the dropdown button is skipped and is not read by the accessibility tools meaning it does not need accessibility features.
